src/govuk/components/accordion/accordion.yaml

```
@@ -169,6 +185,29 @@ examples:
         content:
           html: <a class="govuk-link" href="#">Link B</a>
 
+- name: with translations
+  data:
+    id: with-translations
+    hideAllSectionsHtml: Collapse all sections
+    showAllSectionsHtml: Expand all sections
+    hideSectionHtml: |
+      Collapse <span class="govuk-visually-hidden">this section</span>
+    showSectionHtml: |
```

```suggestion
    showSectionHtml: |-
```

Incredibly minor thing, but the HTML for the translated accordion example ends up with endquotes on newlines:

```html
<div
  class="govuk-accordion"
  data-module="govuk-accordion"
  id="with-translations" data-i18n.hide-all-sections="Collapse all sections" data-i18n.hide-section="Collapse &lt;span class=&quot;govuk-visually-hidden&quot;&gt;this section&lt;/span&gt;
" data-i18n.show-all-sections="Expand all sections" data-i18n.show-section="Expand &lt;span class=&quot;govuk-visually-hidden&quot;&gt;this section&lt;/span&gt;
">
```

---

Nice catch 👍 Thanks